base/reflection.jl

```diff
@@ -542,14 +543,14 @@ code_llvm(f::ANY, types::ANY=Tuple) = code_llvm(STDOUT, f, types)
 code_llvm_raw(f::ANY, types::ANY=Tuple) = code_llvm(STDOUT, f, types, false)
 
 """
-    code_native([io], f, types)
+    code_native([io], f, types, [asm_variant])
 
 Prints the native assembly instructions generated for running the method matching the given
-generic function and type signature to `io` which defaults to `STDOUT`.
+generic function and type signature to `io` which defaults to `STDOUT`.Switch assembly syntax using asm_variant:[att|intel].Output is AT&T syntax by default.
```

Comment:
From this, it is not clear that asm_variant should be given as a string. Maybe: ``The assembly syntax to use can be controlled with `asm_variant`; use "att" for AT&T style (default) or "intel" for Intel style.``